content/sensu-go/6.4/release-notes.md

```diff
@@ -85,6 +86,44 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.4.0 release notes
+
+**June 23, 2021** &mdash; The latest release of Sensu Go, version 6.4.0, is now available for download. 
+
+<!--// synopsis placeholder //-->
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.4.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][207]) The system information modal in the Web UI now includes license expiration information, accessed via the `CTRL + .` keyboard shortcut.
+- Added `darwin/arm64` to the list of platforms we build sensu-agent & sensuctl for.
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][207]) Remove extraneous auto-completion suggestions.
+- ([Commercial feature][207]) Add the ability to configure certain parameters per page in the Web UI.
+- Added etcd-log-level configuration flag for setting the log level of the embedded etcd server.
+- Added API key authentication support to sensuctl.
+- Added wait flag to the `sensu-backend init` command which toggles waiting indefinitely for etcd to become available.
+- Added `sensu_go_keepalives` prometheus counter.
+- Upgraded Go version from `1.13.15` to `1.16.5`.
+- Upgraded Etcd version from `3.3.22` to `3.5.0`.
```

Comment:
```suggestion
- Upgraded etcd version from `3.3.22` to `3.5.0`.
```

Reply:
Move https://etcd.io/docs/v3.5/metrics/etcd-metrics-latest.txt to a link at the bottom of the file